addons/project/static/src/js/tours/project.js

```diff
@@ -113,7 +114,7 @@ registry.category("web_tour.tours").add('project_tour', {
 }, {
     trigger: ".o_field_widget[name='user_ids'] input",
     extra_trigger: '.o_form_project_tasks',
-    content: _t("Assign a responsible to your task"),
+    content: _t("Create a Assignee responsible to your task"),
```

Comment:
Hello, 

- Earlier when we create the task assignee was default set in the task
- So the step to set the assignee was not in the tour
- But now we don't set assignee default to task
- Most of the time you have seen in the tour that we set the Many2one field and the focus comes out of the input
- But this doesn't happen in the Many2mnay field
- Assignee was Many2many field so when we set assignee the focus doesn't come out of the input
- We need to write in the Many2many field to make focus comes out of the input
- So that's why I wrote to "create an assignee"

Thanks

Reply:
Hello, sorry but I don't fully understand what you are trying to explain :sweat_smile: 
We cannot change the step description without changing the step itself.
The tour seems to be working fine, what issue are you trying to solve?

Reply:
> Why this change?

I add log note in task. please look at.thanks